src/Engine/Combat/StrikeParams.java

```diff
@@ -115,14 +127,22 @@ protected StrikeParams(StrikeParams other)
     this.attackPower = other.attackPower;
     this.luckRolled = other.luckRolled;
     this.isCounter = other.isCounter;
+    aw1Luck = attacker.CO.aw1Combat;
+    this.attackerDamageMultiplier = other.attackerDamageMultiplier;
+    this.defenderDamageMultiplier = other.defenderDamageMultiplier;
 
     this.targetCoord = other.targetCoord;
   }
 
   public int calculateDamage()
   {
     int luckDamage = getLuck();
-    final int rawDamage = baseDamage * attackPower * damageMultiplier / 100 / 100;
+    if( aw1Luck && isCounter )
+      luckDamage = 0;
+    final int rawDamage = baseDamage * attackPower * attackerDamageMultiplier / 100 / 100;
+    int hpScalingDamage = rawDamage;
+    if( !aw1Luck )
+      hpScalingDamage += luckDamage;
```

Comment:
Changing luck damage to zero for counters is part of what FFFox talked about, but he also said that luck damage in AW1 does not scale with HP, but in later games it does. Are you capturing that also (or don't care)?

Reply:
Yup. It's a few lines down in the damage formula.

Reply:
I threw a nice doc comment on the `aw1Luck` variable.